Loadouts/common.hpp

```diff
@@ -6,9 +6,9 @@
 #define RADIO_LR "ACRE_PRC117F"
 
 // GEAR
-#define BASE_MEDICAL "ACE_elasticBandage:4","ACE_packingBandage:2","ACE_tourniquet","ACE_splint"
-#define MEDIC_MEDICAL RADIO_MR,"ACE_elasticBandage:25","ACE_packingBandage:15","ACE_epinephrine:10","ACE_salineIV:2","ACE_salineIV_500:4","ACE_salineIV_250:8","ACE_morphine:16","ACE_tourniquet:6","ACE_splint:12","ACE_adenosine:10"
-#define PL_MEDIC_MEDICAL RADIO_MR,"ACE_elasticBandage:15","ACE_packingBandage:10","ACE_epinephrine:10","ACE_salineIV:8","ACE_salineIV_500:8","ACE_salineIV_250:8","ACE_morphine:5","ACE_tourniquet:3","ACE_splint:6","ACE_adenosine:10"
+#define BASE_MEDICAL "ACE_elasticBandage:4","ACE_packingBandage:2","ACE_tourniquet:2","ACE_splint"
+#define MEDIC_MEDICAL RADIO_MR,"ACE_elasticBandage:25","ACE_packingBandage:15","ACE_epinephrine:10","ACE_salineIV:2","ACE_salineIV_500:4","ACE_salineIV_250:8","ACE_morphine:16","ACE_tourniquet:6","ACE_splint:12","ACE_adenosine:10","potato_naloxone:3","ACE_painkillers:2"
```

Comment:
may want to set the base higher

Reply:
What?

Reply:
oh lmao for painkillers i meant. 2 painkiller items => 10 pills for medics. one pill per person in a reduced squad, not enough for a full squad

Reply:
gotcha. I didn't want to go too crazy with amount of pills if each person is going to have their own 2 (or, originally, if an FTL also had their own bottle of 10)

i'll see if the other idea gets implemented and bump this up if it doesn't

Reply:
other idea was implemented, added pkblister to each rifleman instead of upping the medic's base loadout amt